aesara/tensor/basic_opt.py

```diff
@@ -2472,384 +2303,6 @@ def local_upcast_elemwise_constant_inputs(fgraph, node):
 # Subtensor opts #
 ##################
```

Comment:
These comment lines can be removed

Reply:
@ricardov94, thanks for the suggestion. What’s the best way to change a previous commit in this case? Should I use `git reset HEAD~` or something like that?

Reply:
You can make a new commit and then squash it with the old one in interactive rebase

Reply:
Thanks will do. I'll also fix the error that the tests are reflecting.